content/events/16-17/spelenmetfpgas.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Spelen met FPGAs
+description: Sigasi geeft een introductie tot FPGAs
+time: 27-04-2017
+created_at: 12-04-2017
+---
+FPGAs zijn hipper dan ooit. Alle grote spelers gebruiken deze flexibele chips om een specifieke voorsprong te krijgen op hun concurrenten. Toch zijn FPGAs nog steeds vrij exotisch en weinig toegankelijk.
```

Comment:
Maybe quickly introduce/explain what FPGA's are specifically? This explanation is very very broad